### ISSUE TITLE
503 Service Temporariy Unavailable 

### ISSUE THREAD

**Issue by osamabinsaleem** [LM-Development/aks-sample#61]
Upon completing all the steps defined [here](https://github.com/LM-Development/aks-sample/blob/main/Samples/PublicSamples/RecordingBot/docs/tutorials/deploy/5-helm.md) I'm getting 503  error when  I hit the url  `https://mybotname.westus.cloudapp.azure.com/calls`.

Upon checking the pods I see this: 
```
kubectl get pods -n recordingbottutorial
NAME                                                             READY   STATUS             RESTARTS         AGE
recordingbottutorial-0                                           0/1     CrashLoopBackOff   35 (39s ago)     164m
recordingbottutorial-1                                           0/1     CrashLoopBackOff   34 (2m37s ago)   164m
recordingbottutorial-ingress-nginx-controller-5bc86dbdd5-xwngh   1/1     Running            0                3h20m
```

and describing a pod gives me this: 
```
kubectl logs recordingbottutorial-0 -n recordingbottutorial
Setup: Starting VC_redist
Setup: Converting certificate
Setup: Installing certificate
Certificate "mybotname.westus.cloudapp.azure.com" added to store.

CertUtil: -importPFX command completed successfully.
Setup: Deleting bindings
Setup: Adding bindings
Setup: Done
---------------------
RecordingBot: booting
fail: RecordingBot.Console[0]
      Unhandled exception in Boot()
      Status Code: 0
      Microsoft.Graph.Communications.Core.Exceptions.ServiceException: Media platform failed to initialize
       ---> System.InvalidOperationException: MediaPlatform needs a system with at least 2 cores for creation
         at Microsoft.Skype.Internal.Bots.Media.InternalMediaPlatform.Initialize(MediaPlatformSettings settings, IConfigurationManager configurationManager, Boolean isTest)
         at Microsoft.Skype.Bots.Media.MediaPlatform.Initialize(MediaPlatformSettings settings, IConfigurationManager configManager, Boolean isTest)
         at Microsoft.Skype.Bots.Media.MediaPlatform.Initialize(MediaPlatformSettings settings)
         at Microsoft.Graph.Communications.Calls.Media.MediaCommunicationsClientBuilderExtensions.SetMediaPlatformSettings(ICommunicationsClientBuilder statefulClientBuilder, MediaPlatformSettings mediaSettings)
         --- End of inner exception stack trace ---
         at Microsoft.Graph.Communications.Calls.Media.MediaCommunicationsClientBuilderExtensions.SetMediaPlatformSettings(ICommunicationsClientBuilder statefulClientBuilder, MediaPlatformSettings mediaSettings)
         at RecordingBot.Services.Bot.BotService.InitializeClient() in C:\src\RecordingBot.Services\Bot\BotService.cs:line 63
         at RecordingBot.Services.Bot.BotService.Initialize() in C:\src\RecordingBot.Services\Bot\BotService.cs:line 51
         at RecordingBot.Services.ServiceSetup.AppHost.Boot(String[] args) in C:\src\RecordingBot.Services\ServiceSetup\AppHost.cs:line 75
```

**Comment by 1fabi0** [comment]
I guess this is my fault writing the tutorial as the series has one physical core ([see](https://github.com/microsoftgraph/microsoft-graph-comms-samples/issues/146#issuecomment-572884677)) so the resolution would be to delete the windows nodepool and add a nodepool with a different series that actually has 2 physical cores

**Comment by 1fabi0** [comment]
The issue in the Tutorial was fixed(#65)
